views/projects.erb

```diff
@@ -1,39 +1,49 @@
-<head>
-<title>Sarah Trowbridge's coding projects.</title>
-<meta charset = "utf-8">
-</head>
+<!DOCTYPE html>
+<html>
+  <head>
+  <title>Sarah Trowbridge's coding projects.</title>
+  <meta charset = "utf-8">
+  </head>
 
-<body>
+  <body>
 
-<h1> Sarah's coding projects </h1>
+  <h1> Sarah's coding projects </h1>
 
-<p>
-I am a digital strategist living in Seattle and a student of <a href="http://www.adadevelopersacademy.org/">Ada Developers Academy</a>,
-cohort 4. From September 2015 through September 2016, I will be learning front-end web development with the end-goal of becoming a software developer.
-Find out about some of my recent projects below</p>
+  <nav>
+      <a href="/index">Home</a>
+      <a href="/blog">Blog</a>
+  </nav>
 
-<strong> <h3> <a href="https://gist.github.com/trowbrsa/d5348a1309c53da32b98">Random Menu Generator:</a> </h3> </strong>
-  <p>A small program built in Ruby that offers you a random list of menu items based on user input.</p>
+  <br>
 
-<img src="menu.png" style="zoom: 0.5" alt="Menu!">
-
-
-<strong> <h3> <a href="https://github.com/trowbrsa/BankAccounts">Bank Accounts:</a> </h3> </strong>
   <p>
-    A short program that involves one module to manage multiple classes that operate based on the numbers initialized when
-    a new object of that class is created.
+  I am a digital strategist living in Seattle and a student of <a href="http://www.adadevelopersacademy.org/">Ada Developers Academy</a>,
+  cohort 4. From September 2015 through September 2016, I will be learning front-end web development with the end-goal of becoming a software developer.
+  Find out about some of my recent projects below
   </p>
 
-<img src="bank.png" alt="Piggy bank!">
+  <strong> <h3> <a href="https://gist.github.com/trowbrsa/d5348a1309c53da32b98">Random Menu Generator:</a> </h3> </strong>
+    <p>A small program built in Ruby that offers you a random list of menu items based on user input.</p>
 
-<h4> View a full list of my coding projects via my <a href="https://github.com/trowbrsa/">Github page.</a></h4>
+  <img src="menu.png" style="zoom: 0.5" alt="Menu!">
 
-</body>
 
-<footer>
-  <p>
-    Follow me on:
-    <a href="http://twitter.com/SarahinSEA">Twitter</a>
-    <a href="www.linkedin.com/in/SarahATrowbridge">LinkedIn</a>
-  </p>
-</footer>
+  <strong> <h3> <a href="https://github.com/trowbrsa/BankAccounts">Bank Accounts:</a> </h3> </strong>
+    <p>
+      A short program that involves one module to manage multiple classes that operate based on the numbers initialized when
+      a new object of that class is created.
+    </p>
+
+  <img src="bank.png" alt="Piggy bank!">
```

Comment:
There is, what appears to be, a local file here but it is not checked in. Is this missing from git?